bin/make-consensus.py

```diff
@@ -96,6 +106,15 @@ def main():
               file=sys.stderr)
         sys.exit(0)
 
+    if args.ivar and not args.bam:
+        print('If --ivar is used, --bam must be too.', file=sys.stderr)
+        sys.exit(0)
+
+    if args.ivarFrequencyThreshold and not args.ivar:
```

Comment:
When you make `ivarFrequencyThreshold` into a float, this will need to be checked with `if args.ivarFrequencyThreshold is not None`.

Reply:
(Which allows some to use `--ivarFrequencyThreshold 0.0`)

Reply:
Done.